### DISCUSSION TITLE
IntellijIDEA plugin config

### DISCUSSION THREAD

**Discussion by doukhahmed** [MCMicS/jenkins-control-plugin#529] [Q&A] [answered]
Hello

How to configure the Jenkins plugin in IntelliJ IDEA to connect with a local Jenkins server?

what are the params for :

Servers address
Jenkins URL
Username
API-Token

**Comment by MCMicS** [comment, marked as answer]
Hi
> Servers address:
Your server address: `http://localhost:8080/jenkins`

> Jenkins URL
If your server address is different than server address this yould be set here. For your case you can leave empty

> Username
you username in jenkins server

> API-Token
your created API Token for access server

see: https://github.com/MCMicS/jenkins-control-plugin#configuration-steps

**Reply by doukhahmed** [reply]
Can i connect with a password instead of using the API-Token ?

**Reply by doukhahmed** [reply]
No valid crumb was included in the request !

**Reply by MCMicS** [reply]
Password is not supported anymore because of the crumb issue. Jenkins itself recommend API token usage
